onnxruntime/test/platform/apple/apple_package_test/Podfile.template

```diff
@@ -15,7 +15,7 @@ if ENV['SKIP_MACOS_TEST'] != 'true'
       # Comment the next line if you don't want to use dynamic frameworks
       use_frameworks!
 
-      platform :osx, '11.0'
+      platform :osx, '13.0'
```

Comment:
this is a MacOS version. we still build with a deployment target of 11.0 for MacOS.

https://github.com/microsoft/onnxruntime/blob/9ef28f092f575ce15bfc40c7663e649e75809c53/tools/ci_build/github/apple/default_full_apple_framework_build_settings.json#L27

Reply:
updated